cmd/influxdb-writer/main.go

```diff
@@ -150,6 +157,21 @@ func makeMetrics() (*kitprometheus.Counter, *kitprometheus.Summary) {
 	return counter, latency
 }
 
+func makeTransformer(cfg config, logger logger.Logger) transformers.Transformer {
+	switch strings.ToUpper(cfg.transformer) {
+	case "SENML":
+		logger.Info("Using SenML transformer")
+		return senml.New(cfg.contentType)
+	case "JSON":
+		logger.Info("Using JSON transformer")
+		return json.New()
+	default:
+		logger.Error(fmt.Sprintf("Can't create transformer: unknown transformer type %s", cfg.transformer))
+		os.Exit(1)
```

Comment:
why exit here?

Reply:
I agree, `SenML` should be also `default`

Reply:
Default is probably defined by ENV constant, maybe it is not bad to report error here and give user / deployment engineer info that ENV var has not be set.

Reply:
Yes, SenML is default and it makes sense to put it here, but the problem is that SenML requires format when instantiation, so defaulting (no env set) would also imply that we set a default format. I'm afraid it's too many implicit default values.

Reply:
Agree,  exit is ok here. Better to exit then trick me to use a format that I didn't want it. It's too critical IMHO